alerts/postgres-gke/README.md

```diff
@@ -0,0 +1,31 @@
+# Alerts for PostgreSQL in GKE
```

Comment:
Is there 1 recommended alert policy for Postgres?

Reply:
Yes; We have 3 planned alert policies for ops-agent, but 2 of them are derived from metrics not included in the prometheus exporter, unfortunately, so they have been dropped here.